example_case_folders/08_floris_only/amr_input.inp

```diff
@@ -0,0 +1,181 @@
+ABL.wall_shear_stress_type                  = "Moeng"
+
+ABL.bndry_file = "/projects/ssc/amr_precursors/b_abl_neutral_lowTI_redo/bndry_files"
+ABL.bndry_io_mode = 1
+ABL.bndry_planes = ylo xlo # I'm (Paul) adding this but not sure if I have to
+ABL.bndry_var_names = velocity temperature tke
+
+
+# Output boundary files
+ABL.bndry_planes = ylo xlo
+ABL.bndry_output_start_time = 7200.0
+ABL.bndry_var_names = velocity temperature tke
+ABL.bndry_output_format = native
+ABL.stats_output_frequency = 1                   
+ABL.stats_output_format = netcdf  
+
+# Whether to use helics
+helics.activated        =   true
+helics.broker_port = 32000
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#        ADAPTIVE MESH REFINEMENT       #
+#.......................................#
+amr.n_cell              = 512 512 128    # Grid cells at coarsest AMRlevel
+amr.max_level           = 0           # Max AMR level in hierarchy 
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              GEOMETRY                 #
+#.......................................#
+geometry.prob_lo        =   0.       0.     0.  # Lo corner coordinates
+geometry.prob_hi        =   5120.  5120.  1280.  # Hi corner coordinates
+geometry.is_periodic  =   0 0 0
+xlo.type = "mass_inflow"
+xlo.density = 1.0
+xlo.temperature = 0.0 # value required but ignored
+xlo.tke = 0.0
+xhi.type = "pressure_outflow"
+ 
+ylo.type = "mass_inflow"
+ylo.density = 1.0
+ylo.temperature = 0.0
+ylo.tke = 0.0
+yhi.type = "pressure_outflow"
+
+# Boundary conditions
+zlo.type =   "wall_model"
+zlo.tke_type = "zero_gradient"
+
+zhi.type =   "slip_wall"
+zhi.temperature_type = "fixed_gradient"
+zhi.temperature = 0.003 # tracer is used to specify potential temperature gradient
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              VERBOSITY                #
+#.......................................#
+incflo.verbose          =   0          # incflo_level
+
+
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              SAMPLING                 #
+#.......................................#
+incflo.post_processing = samplingPlane samplingLine
+
+samplingPlane.output_frequency = 600
+samplingPlane.labels = z_plane
+samplingPlane.fields = velocity temperature
+samplingPlane.z_plane.type = PlaneSampler
+samplingPlane.z_plane.axis1 = 5110 0.0 0.0
+samplingPlane.z_plane.axis2 = 0.0 5110 0.0
+samplingPlane.z_plane.origin = 5.0 5.0 0.0
+samplingPlane.z_plane.num_points = 512 512
+samplingPlane.z_plane.normal = 0.0 0.0 1.0
+samplingPlane.z_plane.offsets = 5.0 85.0 155.0 255.0
+
+
+samplingLine.output_frequency = 1
+samplingLine.labels = z_line
+samplingLine.fields = velocity temperature
+samplingLine.z_line.type = LineSampler
+samplingLine.z_line.num_points = 128
+samplingLine.z_line.start = 5.0 5.0 5.0
+samplingLine.z_line.end = 5.0 5.0 1275.0
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              TURBINES                 #
+#.......................................#
+
+Actuator.type = UniformCtDisk
```

Comment:
We have two definitions of turbines in here that is messing up the turbine definition in the amr_standin.  Should we only allow only one turbine definition, or adjust the way it's read?

Reply:
I'll enforce that the Actuator type is UniformCtDisk for the FLORIS stand-in and remove the JoukowskyDisk information below.

Reply:
I'll remove the JoukowskyDisk definition here.

Reply:
@genevievestarke `read_amr_wind_input()` on `Emulator()` now checks for the specified `actuator_type` and gets the correct `rotor_diameter`. I checked this before removing the JoukowskyDisk definition for the input file and it ran as expected; then I went ahead and removed the JoukowskyDisk in this example.